clean-and-build.sh

```diff
@@ -89,13 +109,19 @@ done
 roles=("caseworker-sscs" "caseworker-sscs-systemupdate" "caseworker-sscs-anonymouscitizen" "caseworker-sscs-callagent" "caseworker-sscs-judge" "caseworker-sscs-panelmember")
 
 TRY_AGAIN_SECONDS=15
+ATTEMPTS=0
 for role in "${roles[@]}"
 do
   echo "Creating role $role"
   until ./bin/ccd-add-role.sh $role
   do
-    echo "Failed to create role. This is normal, trying again in $TRY_AGAIN_SECONDS seconds"
+    echo "Failed to create role. This might be ok - trying again in $TRY_AGAIN_SECONDS seconds"
```

Comment:
glad you changed this message. It always confused me when something fails and it's telling me it's normal.